docs/guides/unbound.md

```diff
@@ -52,82 +52,154 @@ The first thing you need to do is to install the recursive DNS resolver:
 ```ini
 server:
+    # One thread should be sufficient, can be increased on beefy machines. In reality for most users running on small networks or on a single machine it should be unnecessary to seek performance enhancement by increasing num-threads above 1.
+    num-threads: 1
+
     # If no logfile is specified, syslog is used
     # logfile: "/var/log/unbound/unbound.log"
     verbosity: 0
 
-    port: 5353
+    # Ensure Unbound is running on loopback interface
+    interface: 127.0.0.1@53
+
+    # Use an unassigned port
+    port: 10053
+
     do-ip4: yes
     do-udp: yes
     do-tcp: yes
 
     # May be set to yes if you have IPv6 connectivity
     do-ip6: no
 
-    # Use this only when you downloaded the list of primary root servers!
-    root-hints: "/var/lib/unbound/root.hints"
-
-    # Trust glue only if it is within the servers authority
-    harden-glue: yes
-
-    # Require DNSSEC data for trust-anchored zones, if such data is absent, the zone becomes BOGUS
-    harden-dnssec-stripped: yes
-
-    # Don't use Capitalization randomization as it known to cause DNSSEC issues sometimes
-    # see https://discourse.pi-hole.net/t/unbound-stubby-or-dnscrypt-proxy/9378 for further details
-    use-caps-for-id: no
-
-    # Reduce EDNS reassembly buffer size.
-    # Suggested by the unbound man page to reduce fragmentation reassembly problems
-    edns-buffer-size: 1472
-
-    # Perform prefetching of close to expired message cache entries
-    # This only applies to domains that have been frequently queried
-    prefetch: yes
-
-    # One thread should be sufficient, can be increased on beefy machines. In reality for most users running on small networks or on a single machine it should be unnecessary to seek performance enhancement by increasing num-threads above 1.
-    num-threads: 1
-
-    # Ensure kernel buffer is large enough to not lose messages in traffic spikes
-    so-rcvbuf: 1m
-
-    # Ensure privacy of local IP ranges
+    # Access control to prevent access from non local network devices
+    # Block all
+    access-control: 0.0.0.0/0 refuse
+    access-control: ::0/0 refuse
+    # Allow lo0
+    access-control: 127.0.0.0/8 allow
+    access-control: ::1 allow
+    # Allow lan
+    access-control: 192.168.0.0/16 allow
+    access-control: 172.16.0.0/12 allow
+    access-control: 10.0.0.0/8 allow
+    access-control: fd80:1fe9:fcee::/48 allow
+
+    # Prints one line per query to the log, making the server (significantly) slower
+    #log-queries: no
+
+    # Prevent banner disclosure
+    hide-identity: yes
+    hide-version: yes
+    # Refuse trustanchor.unbound queries
+    hide-trustanchor: yes
+
+    # The maximum dependency depth that Unbound will pursue in answering a query (reduced memory)
+    target-fetch-policy: "2 1 0 0 0 0"
+
+    harden-short-bufsize: yes
+    harden-large-queries: yes
+
+    # Use 0x20-encoded random bits in the query to foil spoof attempts
+    use-caps-for-id: yes
```

Comment:
```diff
-    # Don't use Capitalization randomization as it known to cause DNSSEC issues sometimes
-    # see https://discourse.pi-hole.net/t/unbound-stubby-or-dnscrypt-proxy/9378 for further details
-    use-caps-for-id: no
+    # Use 0x20-encoded random bits in the query to foil spoof attempts
+    use-caps-for-id: yes
```
Why do you think think this is no issue anymore?

Reply:
By reading the latest discussions on the NLnet Labs mailing list it seems that websites that weren't compliant fixed the issues, I've used use-caps-for-id: yes for a long time with no problems. But the original comment to Pi-hole Discourse could be a good idea.